Doc/source/optional.rst

```diff
@@ -1,142 +1,112 @@
+To use the latest Unicode version, install:
 
-*Extra:* ``unicode``
+- `unicodedata2 <https://pypi.python.org/pypi/unicodedata2>`__: backport of `unicodedata` for Python 3.x, updated to unicode 14.0. Not necessary with Python 3.11.
 
+*Extra:* `unicode`
 
 :py:mod:`fontTools.varLib.interpolatable`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------
 
-Module for finding wrong contour/component order between different masters.
-It requires one of the following packages in order to solve the so-called
-"minimum weight perfect matching problem in bipartite graphs", or
-the Assignment problem:
+Module for resolving contour or component order between different masters. Requires following packages to solve 'minimum weight perfect matching problem in bipartite graphs':
 
-* `scipy <https://pypi.python.org/pypi/scipy>`__: the Scientific Library for Python, which internally
-  uses `NumPy <https://pypi.python.org/pypi/numpy>`__ arrays and hence is very fast;
-* `munkres <https://pypi.python.org/pypi/munkres>`__: a pure-Python module that implements the Hungarian
-  or Kuhn-Munkres algorithm.
-
-*Extra:* ``interpolatable``
+- `scipy <https://pypi.python.org/pypi/scipy>`__: Scientific Library for Python, uses `NumPy <https://pypi.python.org/pypi/numpy>`__ arrays for performance;
+- `munkres <https://pypi.python.org/pypi/munkres>`__: implementation of the Hungarian or Kuhn-Munkres algorithm.
 
+*Extra:* `interpolatable`
 
 :py:mod:`fontTools.varLib.plot`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Module for visualizing DesignSpaceDocument and resulting VariationModel.
+----------------------
 
-* `matplotlib <https://pypi.org/pypi/matplotlib>`__: 2D plotting library.
+Module for visualizing DesignSpaceDocument and resulting VariationModel. Requires:
 
-*Extra:* ``plot``
+- `matplotlib <https://pypi.org/pypi/matplotlib>`__: 2D plotting library.
 
+*Extra:* `plot`
 
 :py:mod:`fontTools.misc.symfont`
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Advanced module for symbolic font statistics analysis; it requires:
+----------------------
 
-* `sympy <https://pypi.python.org/pypi/sympy>`__: the Python library for symbolic mathematics.
+Advanced module for symbolic font statistics analysis. Requires:
 
-*Extra:* ``symfont``
+- `sympy <https://pypi.python.org/pypi/sympy>`__: Python library for symbolic mathematics.
 
+*Extra:* `symfont`
 
 :py:mod:`fontTools.t1Lib`
-^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------
 
-To get the file creator and type of Macintosh PostScript Type 1 fonts
-on Python 3 you need to install the following module, as the old ``MacOS``
-module is no longer included in Mac Python:
+Retrieves information about Macintosh PostScript Type 1 fonts on Python 3. Requires:
 
-* `xattr <https://pypi.python.org/pypi/xattr>`__: Python wrapper for extended filesystem attributes
-  (macOS platform only).
-
-*Extra:* ``type1``
+- `xattr <https://pypi.python.org/pypi/xattr>`__: Python wrapper for extended filesystem attributes, macOS only.
```

Comment:
Feel free to disagree here, but I always worry about ambiguity when saying "macOS only" or the like unless it's totally spelled out. As in, are we saying that the module can only be installed on macOS? Or just that the xattrs are only found on macOS (different question how you'd be accessing those filesystems on another platform, though).

Reply:
(same "only"ness question on the CocoaPen bit below, I suppose, although I think mounting HFS/HFS+ disks is still possible on Linux, so the xattrs stuff *might* be usable there. I'd certainly be happy to learn for sure)